ticdc/troubleshoot-ticdc.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Common Problems and Troubleshooting
+summary: Fix common problems while using TiCDC.
+category: reference
+aliases: ['/docs/dev/reference/tools/ticdc/troubleshoot-ticdc/']
+---
+
+# Common Problems and Troubleshooting
+
+This document includes some common problems while using TiCDC and appropriate solutions. It also summarizes some common operation failures and ways to fix them.
```

Comment:
Adverbial clauses can omit the subject (and the be verb) only if the subject of the dependent (the adverbial clause) is the same as that of the independent clause. I see multiple similar issues in this PR.

Reply:
Thanks for your review. I have added possible subjects (not 100% sure). The subject seemed vague to me in some original sentences, because I didn't get the whole picture of TiCDC.